conf/igenomes.config

```diff
@@ -26,12 +27,30 @@ params {
             intervals             = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/intervals/wgs_calling_regions_Sarek.list"
             known_indels          = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/GATKBundle/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf"
             known_indels_tbi      = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/GATKBundle/{1000G_phase1,Mills_and_1000G_gold_standard}.indels.b37.vcf.idx"
+            known_indels_mills_vqsr     = 'mills,known=false,training=true,truth=true,prior=12 Mills_and_1000G_gold_standard.indels.b37.vcf'
+            known_indels_1000g_vqsr     = '1000G,known=false,training=true,truth=true,prior=10 1000G_phase1.indels.b37.vcf'
             mappability           = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/Control-FREEC/out100m2_hg19.gem"
+            res_1000g             = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/GATKBundle/1000G_phase1.snps.high_confidence.b37.vcf.gz"
+            res_1000g_tbi         = "${params.igenomes_base}/Homo_sapiens/GATK/GRCh37/Annotation/GATKBundle/1000G_phase1.snps.high_confidence.b37.vcf.idx.gz"
+            res_1000g_vqsr        = '1000G,known=false,training=true,truth=true,prior=10 1000G_phase1.snps.high_confidence.b37.vcf.gz'
             snpeff_db             = 'GRCh37.75'
             snpeff_genome         = 'GRCh37'
             vep_cache_version     = 104
             vep_genome            = 'GRCh37'
             vep_species           = 'homo_sapiens'
+
+            // resources for GATK joint germline variant recalibration
+            RESOURCE_SNP = [
+                [ res_1000g, dbsnp ],
+                [ res_1000g, dbsnp_tbi ], 
+                [ res_1000g_vqsr, dbsnp_vqsr ]
+            ]
+            resource_INDEL = [
+                [ known_indels, dbsnp ],
+                [ known_indels_tbi, dbsnp_tbi ], 
+                [ known_indels_mills_vqsr, known_indels_1000g_vqsr, dbsnp_vqsr ]
+            ]
```

Comment:
I like that, but I feel like it should be done in the sarek script or in the joint germline variant calling workflow instead

Reply:
I would then have to use less descriptive names as for hg19 and hg38 the files are slightly different. So the naming convention has to match regardless of the genome

Reply:
what about something like `known_snps` (dbsnp should stay separate because tools like haplotypecaller explicetly want that file)